develocity-gradle-plugin-adapters/build.gradle.kts

```diff
@@ -32,9 +32,3 @@ develocityAdaptersPublication {
     name = "Develocity Gradle API adapters for Develocity and Gradle Enterprise Gradle plugins"
```

Comment:
This is still mentioning Gradle Enterprise. Please update all occurrences to Develocity in this project.